walkthroughs/howto-k8s-cross-cluster/deploy.sh

```diff
@@ -34,7 +34,7 @@ fi
 
 AWS_CLI_VERSION=$(aws --version 2>&1 | cut -d/ -f2 | cut -d. -f1)
 DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
-PROJECT_NAME="appmesh-demo"
+PROJECT_NAME="howto-k8s-cross-cluster"
```

Comment:
This change in namespace name from `appmesh-demo` to `howto-k8s-cross-cluster` needs to be reflected in the README as well